src/test/resources/saReport.txt

```diff
@@ -6,27 +6,27 @@
          Network balance: active generation=607.0 MW, active load=600.0 MW, reactive generation=0.0 MVar, reactive load=200.0 MVar
       + Pre-contingency simulation
          AC load flow complete with NR status 'CONVERGED'
-        + Outer loop Distributed slack on generation
```

Comment:
You don't have `on generation` any more.

Reply:
yes this is on purpose